lib/dialogue-agent/assistant_dispatcher.ts

```diff
@@ -161,11 +164,22 @@ export default class AssistantDispatcher extends events.EventEmitter {
         messages : Message[],
         askSpecial : string|null;
     }> {
+        const state = await this._conversationStateDB.getOne(conversationId).then((row) => {
+            if (row) {
+                return {
+                    history : row.history ? JSON.parse(row.history) : [],
+                    dialogueState : row.dialogueState ? JSON.parse(row.dialogueState) : null,
+                    lastMessageId : row.lastMessageId ? row.lastMessageId : 0
+                };
+            }
+            return undefined;
+        });
```

Comment:
This is inside converse() which is used by the REST endpoint /me/api/converse. We'll need equivalent code in getOrOpenConversation (or a new API) so the usual WebSocket endpoint also uses it.